tox.ini

```diff
@@ -1,12 +1,22 @@
 [tox]
 isolated_build = true
-envlist = py{38,39,310,311,312}-django{32,42}-psycopg2,py312-django42-psycopg3,report
+envlist =
+    py312-django50-psycopg3
+    py{310,311,312}-django50-psycopg2
+    py312-django42-psycopg3
+    py{38,39,310,311}-django42-psycopg2
+    py{38,39,310,311}-django41-psycopg2
+    py{38,39,310}-django32-psycopg2
+    report
+
 
 [testenv]
 install_command = pip install {opts} --no-compile {packages}
 deps =
-    django32: Django>=3.2,<3.3
+    django50: Django==5.0rc1
     django42: Django>=4.2,<4.3
+    django41: Django>=4.1,<4.2
+    django32: Django>=3.2,<3.3
```

Comment:
Weird to leave Django versions in descending order but pscyopg versions in ascending order. I think sticking to ascending order is the least invasive, here and in `envlist`. (`tox-ini-fmt` sorts descending, if that would ever be run on this project.)